articles/iot-edge/how-to-configure-proxy-support.md

```diff
@@ -181,7 +181,7 @@ Enter the following text, replacing **\<proxy URL>** with your proxy server addr
 
 ```ini
 [Service]
-Environment="https_proxy=<proxy URL>"
+Environment=https_proxy=<proxy URL>
```

Comment:
The same syntax is also on lines 135 (for version 1.1 in **iotedge**) and 171 (for version 1.2 in **aziot-edged**). Do those need to be fixed, too, or is this just an issue with **aziot-identityd**?

Reply:
Thanks for the catch @kgremban! Was rushing through and didn't check properly. 

I cloned the repo this time and made sure in all cases where we set the environment in the service file, we don't use quotes. The only other place was here, but it was already good-to-go:
https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/iot-edge/troubleshoot.md#check-the-status-of-the-iot-edge-security-manager-and-its-logs

Reply:
@and-rewsmith have you also verified that the lack of quotes is not problematic?  If we had quotes in both 1.1 and 1.2 documentation then is this the first time anyone has every hit this problem?  And if both docs are consistent in using quotes then why did it work for 1.2 and not 1.1?

Reply:
I am thinking that it's the first time someone has made an icm from the issue (and it has been fixed in the docs). It probably happened before.

I didn't try it, but I chose to believe css based on what I learned from:
1. https://kinvolk.io/docs/flatcar-container-linux/latest/setup/systemd/environment-variables/
2. https://stackoverflow.com/questions/43278883/how-to-write-systemd-environment-variables-value-which-contains

If you want me to try it out tomorrow anyway I can.